Gems/PhysX/Code/Include/PhysX/EditorHeightfieldColliderRequestBus.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/Component.h>
+
+namespace PhysX
+{
+    // Provides an interface to the Editor Heightfield Collider component
```

Comment:
nit: As this is a public type we should likely use the API style comment (`//!`)